Modules/_operator.c

```diff
@@ -474,6 +474,44 @@ _operator_iconcat_impl(PyObject *module, PyObject *a, PyObject *b)
     return PySequence_InPlaceConcat(a, b);
 }
 
+/*[clinic input]
+_operator.in_ -> bool
+
+    a: object
+    b: object
+    /
+
+Same as a in b.
```

Comment:
```suggestion
Same as: a in b.
```

Reply:
I have had a quick thought about this too. But none of other docstrings have a semicolon like that and adding it just for these 2 doesn't seem right.

I would probably add semicolon to all of them, but I don't think this is my decision to make.